mods/weather/README.txt

```diff
@@ -1,4 +0,0 @@
-Minetest Game mod: weather
```

Comment:
the README.md for this mod is empty

Reply:
So shall I remove the file or leave it as TXT?

Reply:
Well isn't it supposed to have content?

Reply:
Oops I was looking on the wrong local branch; sorry.
I'll fix that.

Reply:
Fixed